### DISCUSSION TITLE
Weird bug that doesn't catch objects not in the view

### DISCUSSION THREAD

**Discussion by txtyash** [LazyVim/LazyVim#1100] [General]
Let's say I have this line:
```ts
if (!(a.length == 3 && new Set(a.flat()).size == 9 && new Set(b.flat()).size == 9 && new Set(c.flat()).size == 9)) {
```
Now let's say the `{` at the end of the line isn't visible on my current window i.e. it doesn't fit in the window. Then I can't run anything like `yt{`. This doesn't happen on `nvim -u NONE`.

I believe this must be a bug since no one would implement anything this bad.

**Comment by dpetka2001** [comment]
For me it correctly copies until `{` (I made the terminal window smaller so that the line wouldn't fit in the window width). But I use `flash.nvim` from the extras. The default LazyVim uses `leap` if I recall correctly. Maybe it's some kind of limitation from `leap`. Have you tried with `flash` instead?

**Reply by txtyash** [reply]
I installed flash and  didn't face the same problem. so leap is the problem here. Thanks

**Reply by folke** [reply]
Best to report this to the flit repo, so the plugin author is aware of the issue

**Reply by txtyash** [reply]
I apologize. It wasn't "leap" it was "flit". I'll report it.
